modules/serverless-upgrade-sub-channel.adoc

```diff
@@ -5,8 +5,8 @@
 [id="serverless-upgrade-sub-channel_{context}"]
 == Upgrading the Subscription Channel
 
-In {ServerlessProductName} versions 1.5.0 and older, the only available Subscription Update Channel was `techpreview`.
-To upgrade to the latest version, you must update the channel to `preview-4.3`.
+In {ServerlessProductName} versions 1.6.0 and older, the only available Subscription Update Channel was `preview-4.3`.
```

Comment:
Actually, the `preview-4.3` channel was only available in version 1.6.0 and not older versions. I'd remove the "and older" from this.